### DISCUSSION TITLE
All about hackathon

### DISCUSSION THREAD

**Discussion by vinzvinci** [EddieHubCommunity/support#854] [General]
I know some of you want to experience hackathon. And it's hard to find a team, so I make a discussion for all of you :)

### Feel free to drop your details and maybe someone want to team up with you.
Examples:
- Hackathon details (optional, only when you have suggestion)
- Your **skills**
- Experience

Have zero experience? that will be okay, all of us starts with zero, so drop your details except personal information :)
> Second option: Go to hackathon event, look for the team and make new friends.

### What is hackathon?
- A hackathon is usually a day long (can be 24hours, 36 hours, 48 hours, etc.) coding competition where software programmers, developers, designers, etc. come together to build and design something cool.

### Who can attend a hackathon?
- Anyone who is interested

### Team size
- Depends, but mostly 4 or 5 max members

### Why you should join a hackathon?
- It forces you to get out of your comfort zone
- It will improve your career development
- Learning opportunity
- It will help you to grow your professional network
- It will turn your business idea into reality
- Host your project into GitHub and get a green square
- And so many more benefits.

More info:
<a href="https://www.quora.com/What-is-a-hackathon-What-do-you-do-in-it-Is-it-a-team-event-If-yes-what-are-team-sizes">https://www.quora.com/What-is-a-hackathon-What-do-you-do-in-it-Is-it-a-team-event-If-yes-what-are-team-sizes</a>

<hr />

#### My tips
- Research the topic and theme before the hackathon
- Plan with your team
- Work with your team
- Prepare for your food before the hackathon ^_^

#### More information from our mentor:  Eddie Jaoude
Link: <a href="https://www.youtube.com/watch?v=ilby1iYT2iY">https://www.youtube.com/watch?v=ilby1iYT2iY</a>

#### Updated hackathon events:
Link: <a href="https://mlh.io/seasons/2021/events">https://mlh.io/seasons/2021/events</a>

![image](https://user-images.githubusercontent.com/73097560/107107516-6c707c80-686c-11eb-9458-2d354c27fcfd.png)

**Comment by eddiejaoude** [comment]
Thanks @vinzvinci , really well written 👍
